salt/__init__.py

```diff
@@ -12,7 +12,7 @@
     )
     sys.stderr.flush()
 
-USE_VENDORED_TORNADO = True
+USE_VENDORED_TORNADO = True if sys.version_info < (3,12) else False
```

Comment:
Isn't it the same as?:
```suggestion
USE_VENDORED_TORNADO = sys.version_info < (3,12)
```